src/table/table.scss

```diff
@@ -16,371 +15,517 @@
     box-sizing: border-box;
   }
 
-  > .iui-tables-body {
+  .iui-header,
+  .iui-row {
```

Comment:
If those style are for header row only, you should define the .iui-header before .iui-row, because else, this style apply to body too. Like this : 
.iui-header, **.iui-header** .iui-row {}